### DISCUSSION TITLE
RAD working but REU's not

### DISCUSSION THREAD

**Discussion by retromarky99** [frntc/RAD#19] [Q&A]
Hi
My RAD appears to be all up and running and I've set it to boot to 16M REU
However when I try and open any of the REU files I get an error saying no data found.

Any help please?
Thanks
RM

**Comment by retromarky99** [comment]
Update. Still same issues. I tried a new SD card, different c64 and different pi 3b (which didn't work at all, I think it;'s an older model). Tried powering pi externally, no difference.

**Comment by futureretrofusion** [comment]
Can you share a high definition photo of your build?

**Comment by retromarky99** [comment]
Hardware or the SD card?

**Reply by retromarky99** [reply]
I'll try that, thanks.
Will report back tomorrow.

**Comment by futureretrofusion** [comment]
Hardware

**Comment by hpingel** [comment]
Can we discuss one example where you want to use the REU emulation? For example, if you put the REU checker in your RAD_PRG subfolder on the SD card. You can get the reu checker PRG from here:
https://www.retro-programming.de/download/reu-checker/

Then in RAD menu enable REU emulation with a size of your choice and launch the REU checker PRG. Does it successfully check the emulated REU memory?

**Reply by retromarky99** [reply]
It doesn't no. Do you think it's a hardware fault?

**Reply by frntc** [reply]
Above you mentioned a RPi 3B -- is it a 3B or 3B+?

Can you do the same test with GeoRAM emulation? (here is a test program: http://www.cbmhardware.de/dlmanager/index.php?id=444 -- it only handles 2MB, but that doesn't matter)

**Comment by retromarky99** [comment]
I get no REU found. Have decided to return the board to the creator and see about a new one.
Thanks for all your help.
